### PR TITLE
Add Goryunova to the student list

### PR DESCRIPTION
**Pull request by GoryunovaA**

### CODE REVIEW

**Comment by ava8katushka** [comment]
Please resolve the conflicts.

**Comment by GoryunovaA** [comment]
Ready

**Comment by ava8katushka** [comment]
There are conflicts unresolved.

File: README.md [hunk 1]
```diff
@@ -10,3 +10,4 @@
 | 0 | Ава Катушка     | Три идиота                             |
 | 1 | Наташа Смирнова | Меланхолия                             |
 | 2 | Веня Стаховский | Интерстеллар                           |
+| 4 | Настя Горюнова  | Унесённые ветром                       |
```
Comment (ava8katushka) [inline README.md]: Number 4 couldn't go after number 2.

**Review by ava8katushka — CHANGES_REQUESTED** [review]

**Comment by GoryunovaA** [comment]
One more time

**Comment by ava8katushka** [comment]
You try to make your file look ready by adding all the people from the list (and it does look ready), but it's not accepted by git because conflicts are unresolved.

**Comment by GoryunovaA** [comment]
Unfortunately, I have no idea what I should do in order to resolve the conflict

**Comment by ava8katushka** [comment]
That's okay. Let's figure out together.
1. Go to https://github.com/ava8katushka/tech_skills_for_pm2/ and see your pull request in the list.
2. Click on it. 
3. You would see the window to resolve the conflicts.

**Comment by GoryunovaA** [comment]
Seems like it's OK now

**Pull request merged**